scripts/install_pmutools.sh

```diff
@@ -25,7 +25,7 @@
 set -e
 
 sudo apt-get install linux-tools-`uname -r` -y
-sudo apt install numactl
+sudo apt install numactl -y
 
 sudo git clone https://github.com/andikleen/pmu-tools /usr/local/pmu-tools
 
```

Comment:
please remove 32-34. 33 and 34 are not needed by the profiler because it uses the full path. 32 must be set up on the host only.

Reply:
done.